exercises/resistor-significants/description.md

```diff
@@ -0,0 +1,16 @@
+If you would build anything with a Raspberry Pi or an Arduino, one of the components you will likely use are resistors. In this exercise, it doesn't matter what a resistor is or how it works, the only thing that matters is that a resistor has a 'value'. And that they are small. So small, that it's hard to read such a value if it were to be actually printed on it. To solve that, fabricators use color codes that denotes the resistor's value. Which is great, but it makes it hard to see the actual value.
```

Comment:
I've tweaked this below. The main changes:
-  I've restructured it to make it flow a little more clearly, and to fill in a few blanks/unknowns that I had as I read it (I found myself googling about resistors to clarify things). 
- I've removed the "or an Arduino" because I don't think we need both and Raspberry Pis are more well know and - I changed the example to brown and green, because the brown/black `10` thing made me think of binary and confused me :)
- I changed "fabricators" to "manufacturers" as I think it's a more common word.
- I specified "resistance value" because I found myself asking "what/why does this have a value" whereas I could immediately visualise "resistance value" as a tangible thing.

```
If you want to build something using a Raspberry Pi, you'll probably use *resistors*. For this exercise, you need to know two things about them:
1. Each resistor has a resistance value. 
2. Resistors are small - so small in fact that if you printed the resistance value on them, it would be impossible to read.  

To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band acts as a digit of a number. For example, if they printed a brown strip (value 1) followed by a green strip (value 5), the resistance value of the resistor would be 15.

In this exercise, you are going to create a helpful program so that you don't have to remember the values of the bands. The program will take two colors as input, and output the correct resistance value.

The band colors are encoded as follows:

- Black: 0
- Brown: 1
- Red: 2
- Orange: 3
- Yellow: 4
- Green: 5
- Blue: 6
- Violet: 7
- Grey: 8
- White: 9
```

Would appreciate everyone's thoughts - especially @kytrinyx and @f3pix. I've not done it as a "suggestion" so I'll leave you to manually put it in the PR if you like ti.

Reply:
This looks good to me! (I know nothing about resistors)

Reply:
These suggestions overall seem fine to me.

> I changed "fabricators" to "manufacturers" as I think it's a more common word.

How about `makers` instead?

Personally if I didn't know anything about resistors having some basic facts included is helpful, adds some color to the exercise (I know, but pun wasn't intended) , makes the "thing" less nebulous even if the minor information isn't directly relateable it helps me to focus what my mind should be considering while solving the problem; as long as the problem description goes on to indicate which information is essential for completing the exercise.